utils/docker/Dockerfile.centos.7

```diff
@@ -33,19 +35,19 @@ baseurl=${MY_REPO}\n\
 enabled=1\n\
 repo_gpgcheck=0\n\
 gpgcheck=1\n" >> /etc/yum.repos.d/local-centos-group.repo;      \
-      yum -y install --disablerepo=extras                       \
+      sed "s/^mirrorlist_expire=0*/mirrorlist_expire=99999999/" \
+          -i /etc/yum.conf;                                     \
+      yum -y install --disablerepo=base,extras,updates          \
              dnf dnf-plugins-core epel-release-7-11;            \
       rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-7;         \
       yum clean all;                                            \
-      dnf clean all;                                            \
-      dnf config-manager --assumeyes --quiet                    \
-                         --disable base extras epel updates;    \
-      sed "s/^mirrorlist_expire=0*/mirrorlist_expire=99999999/" \
-          -i /etc/yum.conf;                                     \
+      dnf config-manager --assumeyes --quiet --disable epel;    \
```

Comment:
You seem to have dropped the (permanent) disabling of base extras and updates in this update.

Reply:
This is still the case in your latest revision.

Reply:
Fixed in next push.